app/resources/css-default.template

```diff
@@ -196,27 +196,33 @@ input {
   font-size: 13px;
   margin-bottom: .5em;
   max-width: 100%;
+}
+
+/* Show disabled input states with dotted line */
+input:not([type]):disabled {  /* not([type] means default input type */
   border: 0;
   outline: 0;
-  border-bottom: 1px solid #a9a9a9;
+  border-bottom: 1px dotted #a9a9a9;
   background: transparent;
 }
 
-input:focus {
+input:not([type]),  /* not([type] means default input type */
+input[type="text"] {
   border: 0;
   outline: 0;
-  border-bottom: 2px solid #3b78e7;
+  border-bottom: 1px solid #a9a9a9;
+  background: transparent;
 }
 
-/* Show disabled input states with dotted line */
-input:not([type]):disabled {
+input:not([type]):focus,  /* not([type] means default input type */
+input[type="text"]:focus {
   border: 0;
   outline: 0;
-  border-bottom: 1px dotted #a9a9a9;
-  background: transparent;
+  border-bottom: 2px solid #3b78e7;
 }
 
-input[type="file"] {
+/* Remove the focus for photo upload input */
```

Comment:
s/focus/focus frame/ would be clearer. Also s/photo/file/ would be better because this is applied to all file input fields, not limited to photo. We probably don't have any file inputs other than photo, though.